examples/guided.py

```diff
@@ -1,9 +1,9 @@
-import json
 import logging
 import os
 import time
 
 import archinstall
+from archinstall.examples.commons import output_configs
```

Comment:
I would say cross-importing examples is strictly forbidden.
The examples should only serve as an example of *"how would you use archinstall"* type of thing.

So I would say this is a blocker and I won't be able to merge this PR sadly.
If we move the logic of saving configuration to `lib/environment/configuration.py` or similar, that would work. We could also move the loading of configuration to that file.

I don't mind making those changes if you wish, but feel free to change if you have time and/or want to :)